app/views/citizens/identify_types_of_incomes/show.html.erb

```diff
@@ -0,0 +1,17 @@
+<%= page_template page_title: t('.page_heading'), template: :form do %>
+  <%= form_with(model: @legal_aid_application, url: citizens_identify_types_of_income_path, method: :patch, local: true) do |form| %>
+
+    <%= govuk_hint t('.no_income_message'), class: 'govuk-!-padding-bottom-5' %>
+
+    <%= transaction_type_check_boxes(
+          form: form,
+          transaction_types: TransactionType.credits,
+          hints: { benefits: t('.benefits_examples') }
+        ) %>
+
+    <%= next_action_buttons(
+          show_draft: local_assigns.key?(:show_draft) ? show_draft : false,
```

Comment:
why do we need this logic? Shouldn't it be always `false` (which is the default value I believe) because we are on the citizen side?

Reply:
Yes - you are right. I'll change it.

Reply:
In fact, as the default for `next_action_buttons` is to set it false, I don't need a `show_draft` clause here at all!